createChannel.js

```diff
@@ -11,10 +11,12 @@ const r1 = readline.createInterface({
  run().catch(err => console.log(err));
 
 async function run() {
-    const client = new WebClient(config.userSlackToken); // Use the User OAuth Token, not Bot User Oauth Token
+    const client = new WebClient(config.slackToken);
     let res = await client.conversations.list({types: 'public_channel, private_channel'});
     res = res.channels.map((item) => item.name_normalized);
     console.log(res);
+    const val = await client.users.lookupByEmail({email: 'val@karpov.io'});
+    const daniel = await client.users.lookupByEmail({email: 'daniel@meanitsoftware.com'});
     r1.question('What is the name of the channel you wish to create? ', function(name) {
```

Comment:
Fine for now, but in the future please no callbacks :)

Reply:
Didn't realize it at the time but its due to the version of node. v 17 you can save the result to a variable but before that it has to be this.